apps/meteor/app/api/server/lib/rooms.ts

```diff
@@ -146,7 +176,7 @@ export async function findRoomsAvailableForTeams({ uid, name }) {
 
 	const userRooms = Subscriptions.findByUserIdAndRoles(uid, ['owner'], { fields: { rid: 1 } })
 		.fetch()
-		.map((item) => item.rid);
+		.map((item: ISubscription) => item.rid);
```

Comment:
For consistency, this is not a complete ISubscription object, it's `Pick<ISubscription, 'rid'>`